components/atoms/Search/search.tsx

```diff
@@ -142,10 +142,7 @@ const Search = ({
             {suggestionsLabel && suggestions.length > 0 ? <div className="pl-5 pt-4">{suggestionsLabel}</div> : null}
             {suggestions.map((suggestion, index) => (
               <div
-                className={clsx(
-                  cursor === index && "_cursorActive bg-slate-100",
-                  "px-4 py-2 overflow-hidden break-all hover:bg-light-slate-2"
-                )}
+                className={clsx(cursor === index && "_cursorActive bg-slate-100", "px-4 py-2 hover:bg-light-slate-2")}
```

Comment:
refactor (blocking): Instead of styling each span, configure it for all spans in it's parent.

```suggestion
                className={clsx(cursor === index && "_cursorActive bg-slate-100", "px-4 py-2 hover:bg-light-slate-2", "[&_span]:max-w-[12.3rem]")}
```

Is `md:max-w-[11rem]` required @adiati98? I'm only asking as the value is smaller for a larger screen.

Reply:
@nickytonline let me check it locally and get back to you.

Reply:
@nickytonline I've tested the suggestions locally, committed, and pushed them. We don't need the `md:max-w-[11rem]`.
Below is the screenshot of the result. The blue box is longer repo name is truncated. And the green box is longer organization name, with the whole repo name gets truncated.

![Screenshot 2024-04-04 201804](https://github.com/open-sauced/app/assets/45172775/18d7eb57-c39c-445e-a5c2-0e693743bfdb)